mantidproject/mantid
# All .ui files within workbench and mantidqt will now be packaged

**Description of work.**
Change `setup.py.in` in the Workbench and `mantidqt` to package all `.ui` files **inside Python packages** within the `workbench` and `mantidqt`.

If the `ui` files are inside a folder that is _not_ a package, they will not be picked up: `settings/` is the package, `ui` files are in `settings/ui`, which does _not_ have a `__init__.py`, these `ui` files will not be packaged.

**To test:**
Compile and Build the installer. Check that `ui` files are present within the generated `_CPack_Packages` folder.

Tested only on Windows.



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
